.dockerignore

```diff
@@ -0,0 +1 @@
+**/node_modules
```

Comment:
Maybe copy stuff from `.gitignore` and mention that they should be kept in sync?